docs/page-models.md

```diff
@@ -6,3 +6,4 @@ If creating new page models, make sure they also inherit from the following
 
 1. the `SocialFields` mixin, so that social image and text fields are added to the 'promote' tab.
 2. the `ColourThemeMixin`, so that a colour theme can be specified for the page.
+3. the `ContactMixin`, so that a contact can be specified for the page's footer.
```

Comment:
It would be great to update the docs to add a page about the colour themes, and a page about the contact mixins, with a bit more detail on how they function. If you haven't already, do add instructions on the contact settings in the footer to the editor's guide.